src/ScriptCs.Contracts/IConsole.cs

```diff
@@ -19,5 +23,11 @@ public interface IConsole
         void ResetColor();
 
         ConsoleColor ForegroundColor { get; set; }
+
+        int HorizontalPosition { get; set; }
+
+        int VerticalPosition { get; set; }
+
+        int Width { get; }
```

Comment:
I would like this property added to the IConsole interface also.  It's helpful when formatting output.